source/community.applications/usr/local/emhttp/plugins/community.applications/ca_settings.page

```diff
@@ -143,7 +143,7 @@ _(Enable developer mode)_:
 
 
 &nbsp;
-: <input id='debugging' type='button' onclick='debug();' value='DOWNLOAD LOG' style='display:none;'></input><input type="submit" name="#apply" value="<?tr("Apply");?>" id='Apply'><input type="button" id="DONE" value="<?tr("Done");?>" onclick="done()">
+: <span><input id='debugging' type='button' onclick='debug();' value='DOWNLOAD LOG' style='display:none;'></input></span><span><input type="submit" name="#apply" value="<?tr("Apply");?>" id='Apply'><input type="button" id="DONE" value="<?tr("Done");?>" onclick="done()"></span>
```

Comment:
A single `<span>` to wrap both buttons would likely be fine. Instead of individually wrapping each button.